libs/util/git/src/providers/github/github.service.ts

```diff
@@ -335,7 +335,6 @@ export class GithubService implements GitProvider {
 
         const file: GitFile = {
           content: buff.toString("utf-8"),
-          htmlUrl: item.html_url,
```

Comment:
are you sure that we don't need it for the other providers?

Reply:
The GitFile. htmlUrl doesn't seem to be referenced anywhere apart from in output of getFile. It can be safely removed